### PR TITLE
Add helm registry mods and redirects

### PR DESCRIPTION
**Pull request by ritawatson**
This PR overhauls the Helm Registry doc as follows:
- clarifies the Protected mode option. 
- removes repetition.
- uses How-to layout.
- removes section on Brownfield clusters, which is covered in better detail in Clusters > Brownfield Clusters.
- adds redirects .
- for docs listed under Registries and Packs, fixes capitalization and removes -ing doc titles and metatitles to make them active voice.  

**Question to reviewers:** I'm not sure why the original doc said Helm OCI version 3.7.0 is required. I created  a Prerequisite and included this information in it. Does it belong in this doc - or does it belong in the OCI Registry doc?

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1046#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | c634001c36042fab20394863a57127fbed675dce | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1046/librarium_build/1603501374471933952) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | c634001c36042fab20394863a57127fbed675dce | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1046/librarium_build/1603501374471933953) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | f7b0ee46d8b991f8d6ac11f2ec7379d8d0bf77e0 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1046/librarium_build/1603503136117362688) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Review by tiwarisumit3 — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1046#" title="Author self-approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1046#pullrequestreview-1220066335" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/09-registries-and-packs.md
Comment (karl-cardenas-coding) [inline content/docs/09-registries-and-packs.md:4]:
```suggestion
metaDescription: "Learn about Packs, how to use and combine Packs, and how to create your Pack ."
```
Another attempt at cleaning up the description.

File: content/docs/09-registries-and-packs/6-helm-charts.md
Comment (karl-cardenas-coding) [inline content/docs/09-registries-and-packs/6-helm-charts.md:20]:
```suggestion
Palette also supports creating custom Helm registries. You can add your own public or private Helm registries to Palette's Helm registry.
```
I think we can avoid specifying a specific person and keep it generic. There are other roles besides developers that use Palette. Many of them are system administrators or infrastructure analysts.

File: content/docs/09-registries-and-packs/6-helm-charts.md
Comment (karl-cardenas-coding) [inline content/docs/09-registries-and-packs/6-helm-charts.md:22]:
```suggestion
The Helm Chart registry synchronizes charts with Palette, so you can use them when you create cluster profiles.
```
Let's leave out the console. The console is really the UI and not the driver of logic. So we can just say Palette 😄

File: content/docs/09-registries-and-packs/6-helm-charts.md
Comment (karl-cardenas-coding) [inline content/docs/09-registries-and-packs/6-helm-charts.md:24]:
```suggestion
# Prerequisite
```
This should always be a major heading. That way, users can find it in the right-hand-side navigation menu.

File: content/docs/09-registries-and-packs/6-helm-charts.md
Comment (karl-cardenas-coding) [inline content/docs/09-registries-and-packs/6-helm-charts.md:68]:
```suggestion
# Validation
```
Major heading, per the how-to style guide

File: content/docs/09-registries-and-packs/6-helm-charts.md
Comment (karl-cardenas-coding) [inline content/docs/09-registries-and-packs/6-helm-charts.md:27]:
```suggestion
# Add a Helm Chart Registry to Palette
```
Per how-to style guide.

File: content/docs/09-registries-and-packs/6-helm-charts.md
Comment (karl-cardenas-coding) [inline content/docs/09-registries-and-packs/6-helm-charts.md:31]:
```suggestion
1. Log in to Palette as a Tenant administrator. 
```

File: content/docs/09-registries-and-packs/6-helm-charts.md
Comment (karl-cardenas-coding) [inline content/docs/09-registries-and-packs/6-helm-charts.md:39]: @ritawatson should this be in bold instead of code blocks? It seems like a UI element but I could be wrong.

File: content/docs/09-registries-and-packs/6-helm-charts.md
Comment (karl-cardenas-coding) [inline content/docs/09-registries-and-packs/6-helm-charts.md:46]:
```suggestion
      * Charts are not synchronized with Palette, and you must type Helm chart names and versions when you create Cluster Profiles.
```

**Review by karl-cardenas-coding — APPROVED** [review]
@ritawatson Looks great, just some minor suggestions and questions but nothing blocking this from rolling out after your review my feedback 👍🏻

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1046#pullrequestreview-1220060947" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1046#" title="Author self-approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1046#pullrequestreview-1220066335" title="Approved">tiwarisumit3</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 2eb8d4aa4bbff8e67416ae96344e8ae9879548c9 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1046/librarium_build/1603543401918107648) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | a8315bc026861de4bf7db8fa3b00bc843e6eedfd | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1046/librarium_build/1603543905297502208) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by spectro-prow** [comment]
@ritawatson: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | c47fd3988b90f978b0c1e2eca744bf15675c02e2 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1046/librarium_build/1603544786088759296) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Pull request merged**
